app/controllers/orders_controller.rb

```diff
@@ -1,7 +1,17 @@
+require "#{Rails.root}/lib/shippingwrapper.rb"
+require 'area'
+
 class OrdersController < ApplicationController
 
   # fullfillment page for the merchant/seller
-  # .select{ |order| order.status == 'paid' }
+  # .select{ |order| order.status == "paid" }
+    ORIGIN = {
+      country: "US",
+      state: "WA",
+      city: "Seattle",
+      zip: "98161"
+    }
+
```

Comment:
Normally the constant would be defined in a Model, not in the controller. I think ORIGIN makes sense in the Order model.